packages/paste-core/utilities/flex/__tests__/flex.test.tsx

```diff
@@ -0,0 +1,317 @@
+    const tree = renderer
+      .create(
+        <Theme.Provider>
+          <Flex display="flex" vAlignContent="stretch">
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrand" minWidth="size20" minHeight="size10" />
+            </Flex>
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrandHighlight" minWidth="size20" minHeight="size10" />
+            </Flex>
+          </Flex>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
+
+describe('Horizontal Alignment', () => {
+  it('it should set a justify-content: flex-start property', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider>
+          <Flex display="flex" hAlignContent="left">
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrand" minWidth="size20" minHeight="size10" />
+            </Flex>
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrandHighlight" minWidth="size20" minHeight="size10" />
+            </Flex>
+          </Flex>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should set a justify-content: center property', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider>
+          <Flex display="flex" hAlignContent="center">
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrand" minWidth="size20" minHeight="size10" />
+            </Flex>
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrandHighlight" minWidth="size20" minHeight="size10" />
+            </Flex>
+          </Flex>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should set a justify-content: flex-end property', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider>
+          <Flex display="flex" hAlignContent="right">
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrand" minWidth="size20" minHeight="size10" />
+            </Flex>
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrandHighlight" minWidth="size20" minHeight="size10" />
+            </Flex>
+          </Flex>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should set a justify-content: space-around property', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider>
+          <Flex display="flex" hAlignContent="around">
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrand" minWidth="size20" minHeight="size10" />
+            </Flex>
+            <Flex>
+              <Box backgroundColor="colorBackgroundBrandHighlight" minWidth="size20" minHeight="size10" />
+            </Flex>
+          </Flex>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
We should replace `toMatchSnapshot` with an actual check to see if the property was set.  you can check the button tests to see how

Reply:
I'm seeing `toMatchSnapshot` in the button tests as well. What am I missing there?

Reply:
Scroll way down lol, there's a different method we're using on aria- attributes

Reply:
Unit tests added.